build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.2'
-    }
+	}
```

Comment:
Hm, whitespace change in an otherwise space file?

-[Unknown]

Reply:
Uh, not sure what happened. I think my Android Studio is misconfigured. I don't normally use it for editing but occasionally for these Android-specific things.